keyboards/pegasus/readme.md

```diff
@@ -0,0 +1,15 @@
+# Pegasus
+
+![pegasus](https://i.imgur.com/6MKJN7w.jpg)
+
+Pegasus is a 40% keyboard with a 12.75u "WKL" layout with 0.75u blockers. 
+
+* Keyboard Maintainer: [melonbred](https://github.com/melonbred)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: Links to where you can find this hardware
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Thank you for your review! I have added this to readme.md